.env.example

```diff
@@ -0,0 +1,173 @@
+# # # # # # # # # # # # # # # # # # #
```

Comment:
Is this file automatically generated from the source code?

Reply:
@twd2 It seems that is not easy to generate .env from codes. We cannot get such as name, help and default value of arguments from ArgumentParser.

Arguments won't change often and many software write examples for `.env` manually as well, so I suggest just keeping this.

Reply:
Synchronizing between the source code and this file requires more developers' effort.

But I'm lazy. ^\_^ T\_T

Reply:
@twd2 To be honest, I think command instructions here is not so useful. Just let them know the relationship between command-line arguments and environment variables, add prefix `VJ_` and replace `_` to `-`, is enough. The instruction here is just for a quicker start, just a few commonly used arguments is fine.

Hmm... About synchronizing, I believe the argument list won't be changed often.